Gamecube/menu/MenuContext.cpp

```diff
@@ -55,7 +55,10 @@ MenuContext::MenuContext(GXRModeObj *vmode)
 	menu::Gui::getInstance().addFrame(configureButtonsFrame);
 
 	menu::Focus::getInstance().setFocusActive(true);
-	setActiveFrame(FRAME_MAIN);
+	
+	// Don't show the menu if we autoboot a game.
+	if (!strlen(&AutobootROM[0]))
```

Comment:
```c
if (!AutobootROM[0]) { ... }
```

Reply:
i would try it later and tell the results.